docs/CHANGELOG.rst

```diff
@@ -13,12 +13,19 @@ Unreleased
 
 Added
 -----
+- Add bcftools version 1.14 to resolwebio/common Docker image
 
 Changed
 -------
 - **BACKWARD INCOMPATIBLE:** Rewrite processes
   ``workflow-bbduk-star-featurecounts-qc-single`` and
   ``workflow-bbduk-star-featurecounts-qc-paired`` to Python
+- Pin R and R package versions in resolwebio/common Docker image
```

Comment:
It might make more sense to group all changelog entries related to the resolwebio/common image under one entry.